data/map planets.txt

```diff
@@ -2649,6 +2712,16 @@ planet Memory
 		threshold 4000
 		fleet "Small Deep Security" 16
 
+planet "Mendez Station"
+	attributes ccor station
+	landscape land/station20
+	music ambient/machinery
+	description `Mendez Station is the closest object to Alpherg's twin stars. Built by the CCOR as part of its efforts to build self-sufficiency in the Outer Rim, the station's gigantic ramscoops collect the system's solar wind and refine it into hyperdrive fuel.`
```

Comment:
```suggestion
	description `Mendez Station is the closest object to Alpherg's twin stars. Built by the CCOR as part of its efforts to engender self-sufficiency in the Outer Rim, the station's gigantic ramscoops collect charged particles from the system's solar winds and refine them into hyperdrive fuel.`
```
`built` followed by `build` feels repetitive

Reply:
Maybe `develop self sufficiency` or `develop autarky`.

Reply:
Could also keep `build`, but then change it to this 
```suggestion
	description `Mendez Station is the closest object to Alpherg's twin stars. Constructed by the CCOR as part of its efforts to build self-sufficiency in the Outer Rim, the station's gigantic ramscoops collect charged particles from the system's solar winds and refine them into hyperdrive fuel.`
```